tests/data_context/test_base_data_context.py

```diff
@@ -213,3 +218,57 @@ def test_delete_expectation_suite_with_cloud_enabled_context_uses_cloud_impl(
 
     context.delete_expectation_suite("my_expectation_suite")
     assert mock_cloud_delete_expectation_suite.call_count == 1
+
+
+@pytest.mark.cloud
+@mock.patch("great_expectations.data_context.DataContext._save_project_config")
+def test_get_validator_with_cloud_enabled_context_saves_expectation_suite_to_cloud_backend(
+    mock_save_project_config: mock.MagicMock,
+) -> None:
+    """
+    What does this test do and why?
+
+    Ensures that the Validator that is created through DataContext.get_validator() is
+    Cloud-enabled if the DataContext used to instantiate the object is Cloud-enabled.
+
+    Saving of ExpectationSuites using such a Validator should send payloads to the Cloud
+    backend.
+    """
+    context = DataContext(ge_cloud_mode=True)
+
+    # Create a suite to be used in Validator instantiation
+    suites = context.list_expectation_suites()
+    expectation_suite_ge_cloud_id = suites[0].ge_cloud_id
+
+    context.create_expectation_suite(
+        "my_test_suite",
+        ge_cloud_id=expectation_suite_ge_cloud_id,
+        overwrite_existing=True,
+    )
+
+    # Grab the first datasource/data connector/data asset bundle we can to use in Validation instantiation
+    datasource = tuple(context.datasources.values())[0]
+    datasource_name = datasource.name
+
+    data_connector = tuple(datasource.data_connectors.values())[0]
+    data_connector_name = data_connector.name
+
+    data_asset_name = tuple(data_connector.assets.keys())[0]
+
+    batch_request = BatchRequest(
+        datasource_name=datasource_name,
+        data_connector_name=data_connector_name,
+        data_asset_name=data_asset_name,
+    )
+
+    # Create Validator and ensure that persistence is Cloud-backed
+    validator = context.get_validator(
+        batch_request=batch_request,
+        expectation_suite_ge_cloud_id=expectation_suite_ge_cloud_id,
+    )
```

Comment:
In a previous iteration of this test, we were also checking if the `DataContext` associated with the `Validator` was a `CloudDataContext`. Just wanting to check : is this test no longer necessary?

Reply:
Rather that checking for a specific implementation detail, I opted to check that a PUT request was made (test behavior instead of specific attr).

Reply:
Great :) Thank you for the clarification. I do like this MUCH better @cdkini